nixos/lib/testing-python.nix

```diff
@@ -103,7 +103,12 @@ rec {
               {
                 virtualisation.qemu.package = qemu_pkg;
               }
-            );
+            ) ++ [(
+              {
+                # Ensure we do not use aliases.
```

Comment:
This should actually only happen in case of Nixpkgs NixOS tests, and not in case users use the testing framework outside Nixpkgs. Any idea how to do that?

Reply:
That is a tradeoff which is worth it imo

Reply:
Well you **could** use `mkDefault`

Reply:
Do we have to use the `nixpkgs.config` options? Do these even work with flakes? Can't this be set (somehow conditionally) on importing the nixpkgs that are used here?

Reply:
No